charts/posthog/templates/clickhouse-operator/configmap.yaml

```diff
@@ -169,26 +233,35 @@ data:
         </logger>
     </yandex>
 
-  03-clickhouse-querylog.xml: |
+  01-clickhouse-03-query_log.xml: |
     <yandex>
-    <query_log replace="1">
-        <database>system</database>
-        <table>query_log</table>
-        <engine>Engine = MergeTree PARTITION BY event_date ORDER BY event_time TTL event_date + interval 30 day</engine>
-        <flush_interval_milliseconds>7500</flush_interval_milliseconds>
-    </query_log>
-    <query_thread_log remove="1"/>
+        <query_log replace="1">
+            <database>system</database>
+            <table>query_log</table>
+            <engine>Engine = MergeTree PARTITION BY event_date ORDER BY event_time TTL event_date + interval 30 day</engine>
+            <flush_interval_milliseconds>7500</flush_interval_milliseconds>
+        </query_log>
+        <query_thread_log remove="1"/>
     </yandex>
-  04-clickhouse-partlog.xml: |
+
+  01-clickhouse-04-part_log.xml: |
     <yandex>
-    <part_log replace="1">
-        <database>system</database>
-        <table>part_log</table>
-        <engine>Engine = MergeTree PARTITION BY event_date ORDER BY event_time TTL event_date + interval 30 day</engine>
-        <flush_interval_milliseconds>7500</flush_interval_milliseconds>
-    </part_log>
+        <part_log replace="1">
+            <database>system</database>
+            <table>part_log</table>
+            <engine>Engine = MergeTree PARTITION BY event_date ORDER BY event_time TTL event_date + interval 30 day</engine>
+            <flush_interval_milliseconds>7500</flush_interval_milliseconds>
+        </part_log>
     </yandex>
+
 ---
+
+# Template Parameters:
+#
+# NAME=etc-clickhouse-operator-templatesd-files
+# NAMESPACE=posthog
```

Comment:
should we change these too?

Reply:
Aren't these metadata comments?

Reply:
just checking because you did change `#namespace:` ones